src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -59,15 +75,23 @@ export default function withAnnotations<P extends object>(
             }
         };
 
+        handleOnAnnotator = (annotator: Annotator): void => {
+            this.annotator = annotator;
+        };
+
         handlePreviewDestroy = (): void => {
             this.setState(defaultState);
+            this.annotator = undefined;
         };
 
         render(): JSX.Element {
             return (
-                <AnnotatorContext.Provider value={this.state}>
+                <AnnotatorContext.Provider
+                    value={{ state: this.state, emitActiveChangeEvent: this.emitActiveChangeEvent }}
+                >
                     <WrappedComponent
                         {...this.props}
+                        onAnnotator={this.handleOnAnnotator}
```

Comment:
`handleAnnotator` instead of `handleOnAnnotator`

Reply:
I can follow up on this